### PR TITLE
QPICK-49 Identification Cleanup

### PR DESCRIPTION
**Pull request by alexanderraymartin**

### CODE REVIEW

File: frontEnd/package-lock.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "@types/bootstrap": {
       "version": "3.3.36",
       "resolved": "https://registry.npmjs.org/@types/bootstrap/-/bootstrap-3.3.36.tgz",
-      "integrity": "sha512-Ox4QismJMkIIEx800Potvor1PCFg/VldUkmWBARw33mcJejLfXsRjQowdwr20tOzq73+YxrD40kcHbjWDgOwMA==",
+      "integrity": "sha1-XKC4FlVCc5j+fIcovM3DlQaQk9A=",
```
Comment (thomassimko) [inline frontEnd/package-lock.json]: git ignore this file

File: frontEnd/src/js/models/identification/Point.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    get y(): number {
+        return this._y;
+    }
+
+    set y(value: number) {
+        this._y = value;
+    }
+
+}
```
Comment (thomassimko) [inline frontEnd/src/js/models/identification/Point.ts]: why is this necessary? Just make an IPoint.ts that has { x: number, y:number }

File: frontEnd/src/js/react-components/identification/Identification.tsx [hunk 3]
```diff
@@ -1,161 +1,192 @@
...
+                    <a data-tip data-for="exitTip" className="btn btn-default" style={{fontSize:'30px', margin: '5px', boxShadow:'0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19)'}} href="/#/">Exit</a>
+                    <ReactTooltip id='exitTip' type='info'>
+                        <span style={{fontSize:'20px'}}>{'Exit to home screen, discarding current selections.'}</span>
+                    </ReactTooltip>
                 </div>
             </Column>
             <Column lg={12} md={12} sm={12}>
-                <h1 className="text-center"> <a href="#">Need a Refresher?</a></h1>
+                <h1 data-tip data-for="informationTip" className="text-center"> <a href="#">Need a Refresher?</a></h1>
+                <ReactTooltip id='informationTip' type='info'>
```
Comment (thomassimko) [inline frontEnd/src/js/react-components/identification/Identification.tsx]: kind of confused what is going on here.  Would help to see in person

File: frontEnd/src/js/models/identification/Point.ts (hunk 2 above)
Comment (thomassimko) [reply frontEnd/src/js/models/identification/Point.ts]: its not Java you can use json instead of objects

File: frontEnd/src/js/react-components/identification/Identification.tsx (hunk 3 above)
Comment (mdavis60) [reply frontEnd/src/js/react-components/identification/Identification.tsx]: its a tooltip on a link so you don't have to click on the link..........

Comment (thomassimko) [reply frontEnd/src/js/react-components/identification/Identification.tsx]: Wouldn't that just be an I in a circle that they hover over?

Comment (mdavis60) [reply frontEnd/src/js/react-components/identification/Identification.tsx]: Yes but the idea is that the link would take you to the tutorial page.  The tooltip i guess is our short term solution.  Not sure.

File: frontEnd/src/js/models/identification/IdentificationJSON.ts [hunk 4]
```diff
@@ -33,7 +33,7 @@ export class IdentificationJSON implements IIdentificationJSON{
     }
 
     addPoint(x:number, y:number) {
-        this.points.push(new Point(x, y));
+        this.points.push(IPoint(x, y));
```
Comment (thomassimko) [inline frontEnd/src/js/models/identification/IdentificationJSON.ts]: does this work?

File: frontEnd/src/js/react-components/identification/Identification.tsx (hunk 3 above)
Comment (alexanderraymartin) [reply frontEnd/src/js/react-components/identification/Identification.tsx]: Having an "I" was very ugly imo.  This way you can hover over the link for the instructions or click it to go to a full tutorial page.

Comment (thomassimko) [reply frontEnd/src/js/react-components/identification/Identification.tsx]: training != instructions

Comment (thomassimko) [reply frontEnd/src/js/react-components/identification/Identification.tsx]: weird to have them both in the same place

File: frontEnd/src/js/models/Picture.ts [hunk 5]
```diff
@@ -0,0 +1,39 @@
+export interface IPicture {
+    getId(): number;
+    getFileLocation(): string;
+    toJSON(): any;
+}
+
+export class Picture implements IPicture {
```
Comment (thomassimko) [inline frontEnd/src/js/models/Picture.ts]: This can also just be JSON

File: frontEnd/src/js/models/identification/BerryInformation.ts [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+    setShoulder2Y(value: number);
+
+    getParentPicture():any;
+    setParentPicture(parentPicture: any);
+
+    addPoints(ps:IPoint[]);
+    toJSON(): any;
+}
+
+export class BerryInformation implements IBerry {
```
Comment (thomassimko) [inline frontEnd/src/js/models/identification/BerryInformation.ts]: most of this can just be JSON

File: frontEnd/src/js/models/identification/IdentificationJSON.ts [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    getParentPicture():any {
+        return this.parentPicture;
+    }
+
+    setParentPicture(parentPicture:any) {
+        this.parentPicture = parentPicture;
+    }
+
+    addPoint(x:number, y:number) {
+        this.points.push(IPoint(x, y));
```
Comment (thomassimko) [inline frontEnd/src/js/models/identification/IdentificationJSON.ts]: does this work?

File: frontEnd/src/js/models/identification/IdentificationJSON.ts (hunk 4 above)
Comment (mdavis60) [reply frontEnd/src/js/models/identification/IdentificationJSON.ts]: it does not
any suggestions?

File: frontEnd/src/js/models/identification/IdentificationJSON.ts [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+        if (this.points.length == 3) {
+            this.berries.push(this.makeBerryFromPoints());
```
Comment (thomassimko) [inline frontEnd/src/js/models/identification/IdentificationJSON.ts]: if you make berryInformation a JSON you can add the addBerry login into here
